app/src/main/java/pl/edu/zut/mad/appzut/activities/MainActivity.java

```diff
@@ -3,54 +3,74 @@
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
 import android.support.v7.app.AppCompatActivity;
-import android.view.Menu;
-import android.view.MenuItem;
 
 import pl.edu.zut.mad.appzut.R;
 import pl.edu.zut.mad.appzut.fragments.CalendarFragment;
 import pl.edu.zut.mad.appzut.fragments.ScheduleFragment;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static final String CALENDAR_TAG = "calendar_fragment";
+    private static final String SCHEDULE_TAG = "schedule_fragment";
+    private CalendarFragment calendarFragment = new CalendarFragment();
+    private ScheduleFragment scheduleFragment = new ScheduleFragment();
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        startFragments(savedInstanceState);
+    }
 
+    private void startFragments(Bundle savedInstanceState) {
         if (savedInstanceState == null) {
-            startFragment(ScheduleFragment.class);
+            registerCalendarForScheduleFragment();
+            startFragmentInContainer(calendarFragment, R.id.calendar_container, CALENDAR_TAG);
+            startFragmentInContainer(scheduleFragment, R.id.schedule_container, SCHEDULE_TAG);
```

Comment:
I'm not sure if extracting
```
getSupportFragmentManager().beginTransaction()
    .replace(container, fragment)
    .commit();
```
to separate method is looking better, remember that you can do many `.replace()` calls in transaction. However this is open to discussion.